geography/README.md

```diff
@@ -0,0 +1,150 @@
+
+## Table of Contents
+
+* [General Information](#general-information)
+   * [Versioning](#versioning)
+   * [Response Format](#repsonse-format)
+   * [Authorization](#authorization)
+* [Distribution](#distribution)
+* [Schema](#schema)
+  * [Geography Fields](#geography-fields)
+* [File Format](#file-format)
+* [Endpoints](#endpoints)
+
+## General Information
+
+### Versioning
+
+MDS APIs must handle requests for specific versions of the specification from clients.
+
+Versioning must be implemented as specified in the [Versioning section][versioning].
+
+### Response Format
+
+See the [Responses][responses] and [Error Messages][error-messages] sections.
+
+### Authorization
+
+When making requests, the Geography API expects `provider_id` to be part of the claims in a [JWT](https://jwt.io/) `access_token` in the `Authorization` header, in the form `Authorization: Bearer <access_token>`. The token issuance, expiration and revocation policies are at the discretion of the Agency.
+
+<a name="schema"></a>
+
+## Schema
+
+Placeholder -- schema to be added.  
+
+### Geography Fields
+
+| Name               | Type      | Required/Optional | Description                                                                         |
+| ----------------   | --------- | --- | ----------------------------------------------------------------------------------- |
+| `name`             | String    | Required   | Name of geography                                                                      |
+| `description`      | String    | Optional   | Detailed description of geography                                                                      |
+| `geography_id`     | UUID      | Required   | Unique ID of geography                                                                 |
+| `geography_json`   | UUID      | Required   | The GeoJSON that defines the geographical coordinates.
+| `effective_date`   | [timestamp][ts] | Optional   | The date at which a Geography is considered "live".  Must be at or after `publish_date`.
+| `publish_date`     | [timestamp][ts] | Required   | Timestamp that the policy was published, i.e. made immutable                                             |
+| `prev_geographies` | UUID[]    | Optional   | Unique IDs of prior geographies replaced by this one                                   |
+
+<a name="file-format"></a>
+
+## File format
+
+To use flat files rather than REST endpoints, Geography objects should be stored in `geographies.json`.  The `geographies.json` file will look like the output of `GET /geographies`.  
+
+Example `geographies.json`
+```json
+{
+    "version": "0.4.0",
+    "updated:" "1570035222868",
+    "geographies": [
+        {
+            // GeoJSON 1
+        },
+        {
+            // GeoJSON 2
+        }
+    ]
+}
+```
+
+_Note: A simple tool to validate `geographies.json` will be contributed to the Open Mobility Foundation._
```

Comment:
Should this note be included now? How close to being ready is this tool? It may take some time to contribute to a tool to the OMF.

Reply:
We should discuss.  I don't have a good answer.  Perhaps it's trivial now that @thekaveman made a schema?

Reply:
I think just cut it for now so we can get this into the feature branch in OMF.  The schema helps, and I'm not sure what this simple tool would be otherwise.